debian/rules

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-VERSION := $(shell mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v "\[")
+VERSION := $(shell cat pom.xml |sed '22!d'| cut -d'>' -f2 |cut -d'<' -f1)
```

Comment:
I made a separate PR for 4.4 and master. 4.4 got merged and master didn't.. you can check the PR here:

https://github.com/apache/cloudstack/pull/341

It wasn't merged mostly due to aesthetic reasons, although using native tools instead of maven would provide a faster run and should have less compatibility issues.